includes/Data/Themes.php

```diff
@@ -73,6 +73,18 @@ private static function check_approved( $value ) {
 		 return $value['approved'] === true;
 	}
 
+	 /**
+	  * Get the current theme data like the theme variations and previews per step
+	  *
+	  * @return array
+	  */
+	public static function step_preview_data() {
+		$theme_step_data                                 = Patterns::get_count_of_patterns();
+		$theme_step_data['theme-styles']['previewCount'] = $theme_step_data['theme-styles']['previewCount'] *
```

Comment:
Can we add a property in `get_theme_step_patterns` array under `theme-styles` that says something like `combine => true` and then we check for `combine` in `get_count_of_patterns` and perform the calculation there itself, this will help us scale to more steps that need this combination.